netbox/dcim/api/views.py

```diff
@@ -434,6 +436,13 @@ def render_config(self, request, pk):
         context_data.update(request.data)
         context_data.update({'device': device})
 
+        app_ns = registry['model_features']['custom_fields'].keys()
+        for app in app_ns:
+            context_data.setdefault(app, {})
+            models = apps.get_app_config(app).get_models()
+            for model in models:
+                context_data[app][model.__name__] = model
```

Comment:
@abhi1693 what do you think about moving this logic to `ConfigTemplate.render()`? This would ensure the models are always present in the base template context, and avoid the duplication.

Reply:
I was thinking to refactor this under device but that's a better place to put this in to have more consistent flow of code.

Reply:
After refactoring, should the models be displayed in the context data?